src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,91 @@
 package core.basesyntax;
 
+import java.util.ArrayList;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    static final int DEFAULT_INITIAL_CAPACITY = 16;
+    static final int MAXIMAL_CAPACITY = 1 << 30;
+    static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private int size = 0;
+
+    private ArrayList<Entry<K,V>>[]table = new ArrayList[DEFAULT_INITIAL_CAPACITY];
```

Comment:
Using raw types in Java generics is not safe because it can lead to ClassCastException at runtime. Use the diamond operator to ensure type safety. Change to `new ArrayList<>[DEFAULT_INITIAL_CAPACITY];`

Reply:
But, there is a compilation error while using "new ArrayList<>[DEFAULT_INITIAL_CAPACITY];"